main.py

```diff
@@ -0,0 +1,124 @@
+from protagonist import Protagonist
+from adventurelib import *
+
+# TODO replace the print statement with a function leading to your adventure
+def adv_a():
+    print("Adventure A")
+    return 1
+
+def adv_b():
+    print("Adventure B")
+    return 1
+
+def adv_c():
+    print("Adventure C")
+    return 1
+
+#The Different Advenetures and their names
+ADVENTURES = {
+    "A": "Adventure A",
+    "B": "Adventure B",
+    "C": "Adventure C"
+    }
+
+#The functions related to each adventure
+ADVENTURE_FUNCTIONS = {
+    "A": adv_a,
+    "B": adv_b,
+    "C": adv_c
+}
+
+#This is the primary function that you start at when you begin the game
+def intro():
+    adventures = 1 #we start with only 1 adventure availible
+    say("""
+    Welcome brave adventurer! You have 3 different adventures to depart on. 
+    The adventures must be completed in order! To begin we need to know about you!
+    """)
+    # we create a character with a name and pronouns
+    protag = character_creation()
+    say("Let us begin!")
+    # let our user pick their adventure
+    success = pick_adventure(get_options(adventures))
+    adventures += success #if the protag is successful they get another adventure
+    while True:
+        say("Would you like to go on an adventure?")
+        choice = input("(y/n)")
+        if choice == "y" or "Y":
+            success = pick_adventure(get_options(adventures))
+            if adventures < ADVENTURE_FUNCTIONS.count():
```

Comment:
I believe this should be `len(ADVENTURE_FUNCTIONS)`, not `.count()`

Reply:
Ah yes, both work but len the proper way to check the length of a dictionary in python